### PR TITLE
fix: remove special chars from site_name

### PR DESCRIPTION
**Pull request by jona159**
If the issuer name contains an "@" symbol, the site_name will also contain the "@" which causes the ```set_email_string``` function (```account_adapter.py``` l. 196) to return an invalid email.

### CODE REVIEW

File: apps/issuer/models.py
Comment (timber-they) [inline apps/issuer/models.py:586]: This is not very nice. We should either correct the email function or disallow an `@` sign altogether, I think. If you disagree though, you can merge I guess.

Comment (jona159) [reply apps/issuer/models.py:586]: Makes sense, in another PR i am disallowing problematic characters for the issuer name in the frontend now. However, we currently have an issuer on prod that has an "@" symbol in its name, which we cannot verify for this reason. My idea was to merge this change and revert it once this particular issuer is verified. It's not a good solution but i could not think of a better one, do you have another suggestion?

**Review by YouQam — APPROVED** [review]
I approved it for verifying pending issuer. Please make sure to add a ticket for reverting the changes before merge.

**Pull request merged**
